perception/autoware_detected_object_validation/launch/obstacle_pointcloud_based_validator.launch.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@batuhanbeytekin 
Why did you remove `encoding="UTF-8"` ?

Reply:
I now understand it correctly. I apologize for the confusion.